### DISCUSSION TITLE
Export as binary format (such as postcard)

### DISCUSSION THREAD

**Discussion by N4tus** [tweag/nickel#1133] [Ideas]
As far as I search the docs, there are no binary exports available. If a project writes its config in nickel, then a more machine-friendly export format makes more sense than a human readable (for production use), at least to me. This would probably need to attach meta-data to built-ins, such as numbers, about their exact binary representation, but the meta-data system should be flexible enough to handle these situations.

**Comment by yannham** [comment]
Thanks for the suggestion. Let's move this discussion to a feature request issue! As a preliminary remark, I'm not yet sure it's a good use of metadata, to precise such low-level details irrelevant to people writing and maintaining the configuration. But let's fuel the discussion.
